syntax = "proto3";

package fixbug;

option cc_generic_services = true;

import "resultcode.proto";

message GetFriendListRequest {
    uint32 id = 1;
}

message GetFriendListResponse {
    ResultCode result = 1;
    repeated bytes friends = 2;
}

service GetFriendListSerciveRpc {
    rpc GetFriendList(GetFriendListRequest) returns (GetFriendListResponse);
}